webapp/packages/core-root/src/ServerNodeService.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { Bootstrap, injectable } from '@cloudbeaver/core-di';
+import { Executor, IExecutor } from '@cloudbeaver/core-executor';
+import { GraphQLService, WsSocketConnectedEvent } from '@cloudbeaver/core-sdk';
+
+import { ServerNodeError } from './ServerNodeError';
+import { ServerEventId, SessionEventSource } from './SessionEventSource';
+
+@injectable()
+export class ServerNodeService extends Bootstrap {
```

Comment:
As a suggestion, we could potentially refactor the code to move all logic related to the ServerNode into a separate plugin. This would allow for easier extension and modification of the logic in the future. In this case class should be renamed to the ServerNodeBootstrap